### ISSUE TITLE
[bug?] Error in `python': malloc(): memory corruption

### ISSUE THREAD

**Issue by loackerc** [tensorflow/tensorflow#16248]
Error in `python': malloc(): memory corruption: 0x00000000723f9040

Strangely encountered this error when the training was going, it happened after a certain number of iterations (~7000 iterations with image batch size of 1 using coco dataset).  I have also attached the memory map that showed up after the backtrace.

```
*** Error in `python': malloc(): memory corruption: 0x00000000723f9040 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x777e5)[0x7f9d5715a7e5]
/lib/x86_64-linux-gnu/libc.so.6(+0x8213e)[0x7f9d5716513e]
/lib/x86_64-linux-gnu/libc.so.6(__libc_malloc+0x54)[0x7f9d57167184]
/usr/local/lib/python2.7/dist-packages/numpy/core/multiarray.so(+0x1ec51)[0x7f9d560f0c51]
/usr/local/lib/python2.7/dist-packages/numpy/core/multiarray.so(+0x842c8)[0x7f9d561562c8]
/usr/local/lib/python2.7/dist-packages/numpy/core/multiarray.so(+0x844a4)[0x7f9d561564a4]
/usr/local/lib/python2.7/dist-packages/numpy/core/multiarray.so(+0x84920)[0x7f9d56156920]
/usr/local/lib/python2.7/dist-packages/numpy/core/multiarray.so(+0x85105)[0x7f9d56157105]
/usr/local/lib/python2.7/dist-packages/numpy/core/multiarray.so(+0x11b16d)[0x7f9d561ed16d]
/home/user/tools/../data/coco/PythonAPI/pycocotools/_mask.so(+0x9406)[0x7f9d1f276406]
/home/user/tools/../data/coco/PythonAPI/pycocotools/_mask.so(+0x1c05f)[0x7f9d1f28905f]
python(PyEval_EvalFrameEx+0x615e)[0x4ca15e]
python(PyEval_EvalFrameEx+0x5d8f)[0x4c9d8f]
python(PyEval_EvalCodeEx+0x255)[0x4c2765]
python(PyEval_EvalFrameEx+0x68d1)[0x4ca8d1]
python(PyEval_EvalCodeEx+0x255)[0x4c2765]
python(PyEval_EvalFrameEx+0x68d1)[0x4ca8d1]
python(PyEval_EvalCodeEx+0x255)[0x4c2765]
python(PyEval_EvalFrameEx+0x68d1)[0x4ca8d1]
python(PyEval_EvalCodeEx+0x255)[0x4c2765]
python(PyEval_EvalFrameEx+0x68d1)[0x4ca8d1]
python(PyEval_EvalCodeEx+0x255)[0x4c2765]
python(PyEval_EvalFrameEx+0x6099)[0x4ca099]
python(PyEval_EvalCodeEx+0x255)[0x4c2765]
python(PyEval_EvalCode+0x19)[0x4c2509]
python[0x4f1def]
python(PyRun_FileExFlags+0x82)[0x4ec652]
python(PyRun_SimpleFileExFlags+0x191)[0x4eae31]
python(Py_Main+0x68a)[0x49e14a]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf0)[0x7f9d57103830]
python(_start+0x29)[0x49d9d9]
```

```
System Information
VERSION="16.04.3 LTS (Xenial Xerus)"
VERSION_ID="16.04"
VERSION_CODENAME=xenial

Compiler:
c++ (Ubuntu 5.4.0-6ubuntu1~16.04.4) 5.4.0 20160609

PIPs:
msgpack-numpy (0.4.2)
protobuf (3.5.0.post1)
tensorflow-gpu (1.4.1)
tensorflow-tensorboard (0.4.0rc3)

TensorFlow:
tf.VERSION = 1.4.0
tf.GIT_VERSION = v1.4.0-rc1-11-g130a514
tf.COMPILER_VERSION = v1.4.0-rc1-11-g130a514
Sanity check: array([1], dtype=int32)

Env:
LD_LIBRARY_PATH /usr/lib/x86_64-linux-gnu:/usr/local/lib:/usr/local/cuda/lib64:
DYLD_LIBRARY_PATH is unset

GPU:
TITAN Xp

CUDA Lib:
/usr/local/cuda-8.0/lib64/libcudart.so.8.0.61
/usr/local/cuda-8.0/lib64/libcudart_static.a
/usr/local/cuda-8.0/doc/man/man7/libcudart.7
/usr/local/cuda-8.0/doc/man/man7/libcudart.so.7
```
[memory map.txt](https://github.com/tensorflow/tensorflow/files/1647515/memory.map.txt)

**Comment by poxvoculi** [comment]
Thanks for the report, though I suppose this could be due to causes outside of TensorFlow, such as an error in numpy or python or a transient hardware failure.   I don't think we'll take any action on this unless it's reproducible and very likely due to TF.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by poxvoculi** [comment]
Closing due to inactivity.
